### PR TITLE
Fix StandFunnel test.

### PR DESCRIPTION
**Pull request by dmdashenkov**
Sometimes we had a red label from Travis for _some_ builds. This was a concurrency issue in `StandFunnelShould`. Now everything works fine.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol PTAL

File: server/src/test/java/org/spine3/server/stand/StandFunnelShould.java [hunk 1]
```diff
@@ -197,19 +200,15 @@ private static void checkUpdatesDelivery(boolean isConcurrent, BoundedContextAct
         verify(boundedContext, times(dispatchActions.length)).getStandFunnel();
 
         if (isConcurrent) {
-            await(Given.AWAIT_SECONDS);
+            try {
+                ((ExecutorService) executor).awaitTermination(Given.SEVERAL, TimeUnit.SECONDS);
+            } catch (InterruptedException ignore) {
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/stand/StandFunnelShould.java]: `ignored`.

**Comment by armiol** [comment]
@dmdashenkov please see my comment.

**Comment by dmdashenkov** [comment]
@armiol PTAL

**Comment by armiol** [comment]
@dmdashenkov LGTM.

**Pull request merged**